freespace2/resources/mac/Info.plist.in

```diff
@@ -30,5 +34,7 @@
 	<string>MainMenu</string>
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
+	<key>NSHumanReadableCopyright</key>
+	<string>Copyright 1999 Volition, Inc. &amp; Copyright 2005-2024 The Source Code Project.</string>
```

Comment:
Is 2005 specifically for Mac builds, or is it supposed to be FSO in general?  The SCP started in 2002.

Reply:
It should be FSO in general. It was originally 2005-2010, and I obviously didn't pay enough attention to the start date.

But I'll get that corrected shortly.